### PR TITLE
add hpa for the collector

### PR DESCRIPTION
**Pull request by hodayaGamliel**

### CODE REVIEW

File: overops-collector/README.MD [hunk 1]
```diff
@@ -47,19 +47,24 @@ $ helm show values overops/overops-collector
...
+| `image.pullPolicy`                             | Image pull policy                                                                            | `IfNotPresent`                                       |
+| `service.loadBalancerType`                     | Adds Service Loadbalancer Annotation for required Loadbalancer Types (i.e. "nlb")            | `nil`                                                |
+| `autoscaling.enabled`                          | Enable Horizontal Pod Autoscaling                                                            | `false`                                              |
+| `autoscaling.minReplicas`                      | The minimum allowed replicas      	                                                          | `1`                                                  |
+| `autoscaling.maxReplicas`                      | The maximum allowed replicas                                                                 | `false`                                              |
```
Comment (cseverino789) [inline overops-collector/README.MD:63]: Guessing this should be an integer not false? Also probably 10 would be a better default than 100?

File: overops-collector/values.yaml [hunk 2]
```diff
@@ -79,6 +79,10 @@ resources: {}
 
 autoscaling:
   enabled: false
+  minReplicas: 1
+  maxReplicas: 100
```
Comment (cseverino789) [inline overops-collector/values.yaml:83]: Let's set the max to 10.

**Review by cseverino789 — CHANGES_REQUESTED** [review]
Also per discussion, add the recommend collector resources and update Readme to describe them.

File: overops-collector/README.MD (hunk 1 above)
Comment (hodayaGamliel) [reply overops-collector/README.MD:63]: Done

File: overops-collector/values.yaml (hunk 2 above)
Comment (hodayaGamliel) [reply overops-collector/values.yaml:86]: Done

**Review by cseverino789 — APPROVED** [review]

**Pull request merged**